medusa/tv/series.py

```diff
@@ -351,7 +351,7 @@ def is_sports(self):
     @property
     def network_logo_name(self):
         """Get the network logo name."""
-        return self.network.replace(u'\u00C9', 'e').replace(u'\u00E9', 'e').lower()
+        return self.network.replace(u'\u00C9', 'e').replace(u'\u00E9', 'e').replace(' ', '-').lower()
```

Comment:
My point is: we should remove it as it makes no sense, because: 1) it replaces only one unicode char 2) it tries to replace it twice. 😕

Reply:
About 2: It's not the same character twice.
[`\u00C9`](https://m.unicode-table.com/en/00C9/) [`\u00E9`](https://m.unicode-table.com/en/00e9/)

If we do I'd have to find the network names that originally had it and re-add the correct char instead of `e`.

Reply:
True that, I didn't see the difference 😃 Let's not complicate this further at this point.